grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -642,7 +643,7 @@ void validateGraph() throws GraknValidationException {
         }
     }
 
-    private void submitCommitLogs(Map<Schema.BaseType, Set<String>> concepts){
+    protected void submitCommitLogs(Map<Schema.BaseType, Set<String>> concepts){
```

Comment:
It was when I tried to specialize the implementation for Tinkerpop based graphs, to avoid communicating with Engine. After learning the IN-MEMORY is not the same as Tinkerpop, I forgot to put it back to private. But it's a good idea to do so! If the method is not intended to be overwirtten  it should be private.